stable/traefik/templates/deployment.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 spec:
-  replicas: 1
+  replicas: {{ .Values.replicas }}
```

Comment:
Can we set a default right here in case `replicas` isn't set at all? (i.e. in case someone continues using their _old_, customized `values.yaml` with the updated chart)? Something like this:

```
replicas: {{ .Values.replicas | default 1 }}
```